Services/csv_service.py

```diff
@@ -0,0 +1,33 @@
+import csv
+import codecs
+import requests
+
+from Utils.custom_csv_error import CSVServiceError, CSVServiceHTTPError, CSVServiceEmptyDataError
+
+
+class CSVService:
+
+    def get_csv_data(self, url):
+        response = self._get_url_response(url)
+        csv_data = self._get_csv_content(response)
+        return csv_data
+
+    def _get_url_response(self, url):
+        response = requests.get(url, stream=True)
+        if response.status_code == 200:
```

Comment:
200 is a well know error code but still feels like a [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29)

Can you think of an alternative here that can make things even more clear?

Reply:
We can use an enum of possible status codes or a module only with constants ex HTTP_SUCCESS = 200

Reply:
Indeed, further more one such module is already defined in python https://docs.python.org/3/library/http.html